sciLifeLab_utils/run_assembly_report.py

```diff
@@ -196,29 +196,38 @@ def collect_results_and_report(validation_sample_dir, assemblies_sample_dir,
     BUSCO_lineage = []
     for assembler in assemblers_assemblies:
 
-        row = [assembler]
         #Find which BUSCO data set was used from the evaluation config file
         sample_config_g = os.path.join(validation_sample_dir,
                     assembler, "*_evaluete.yaml")
         sample_config_f = glob.glob(sample_config_g)[0]
         with open(sample_config_f, "r") as f:
             sample_config = yaml.load(f)
-        data_path = sample_config["BUSCODataPath"]
+        try:
+            data_path = sample_config["BUSCODataPath"]
+        except KeyError:
+            print("No BUSCO data found. Assuming BUSCO was not run.")
+            BUSCO_lineage.append(None)
+            continue
         BUSCO_lineage.append(data_path)
 
         #Find the BUSCO metrics from the result file
-        summary_g = os.path.join(validation_sample_dir, assembler, "BUSCO", "run_*", "short_summary_*")
+        summary_g = os.path.join(validation_sample_dir, assembler, "BUSCO", "run_*", "full_table_*")
         if len(summary_g) > 0:
```

Comment:
This line is not part of this PR, but still was it intended to be `if len(glob.glob(summary_a)) >0` ? If not wouldn't it be always `True` as `summary_g` is a string that is defined just above this line 👀